### DISCUSSION TITLE
Project Structure and Ideas

### DISCUSSION THREAD

**Discussion by Evavic44** [Evavic44/rocketmeme#5] [Ideas]
## Project Requirements
Here is a list of all that is required to build this project.

### Stack & Library
- React
- JavaScript
- TypeScript
- NodeJS
- [Redux](https://react-redux.js.org/)
- [Framermotion]()
- [Hasura](https://hasura.io/) 
- [React conffetti](https://www.npmjs.com/package/react-confetti)

### Project Time Estimate
How long will it take to build this project?
**Ans: We have one month that's all that matters. 😅**

## Colors & UI
Primary-color: `#ffcf4b`
Secondary-color: `#003566`
Tertiary-color: `#0066ff`
Others: `#FFE460`

<img src="https://user-images.githubusercontent.com/62628408/156275508-71c805fe-cf8d-4d43-8406-bace236ffc17.png" width="500px">

**Comment by SpiffGreen** [comment]
Please share link to the backend service you suggested we use.

**Reply by Evavic44** [reply]
For the authentication?

**Reply by SpiffGreen** [reply]
Yes

**Reply by Evavic44** [reply]
Are we using a service, taught it's google and GitHub auth?

But the service we could use is https://auth0.com/

**Reply by SpiffGreen** [reply]
Noted

**Comment by Evavic44** [comment]
## Note
Create a new release for every major update.

- [ ] All elements of the editor is in place and working
- [ ] Integrated the API with Hasura GraphQL
- [ ] Implement search and pagination 
- [ ] Fix login functionality
